sklearn/ensemble/bagging.py

```diff
@@ -390,8 +390,9 @@ def _set_oob_score(self, X, y):
         """Calculate out of bag predictions and score."""
 
     def _validate_y(self, y):
-        # Default implementation
-        return column_or_1d(y, warn=True)
```

Comment:
So there should still be a `check_array` call here. And if not, we should delete this method.